src/main/java/gr/cytech/chatreminderbot/rest/controlCases/Control.java

```diff
@@ -33,6 +33,9 @@
     @Inject
     CaseShowVersion caseShowVersion;
 
+    @Inject
+    CaseSetConfigurations caseSetConfigurations;
```

Comment:
Although I think I understand what is the reason behind injecting each handler, but it is slowly getting out of hand. Can you please record a task to find a better way to do this, without requiring injection of each and every possible interaction handler?